tests/roxctl/token-file.sh

```diff
@@ -29,6 +29,7 @@ test_roxctl_cmd() {
 
   # Verify that specifying a token file works.
   if OUTPUT=$(roxctl --insecure-skip-tls-verify --insecure -e "$API_ENDPOINT" \
+    --timeout 2m \
```

Comment:
The default timeout for `roxctl central db` is `1h`. Did you mean to set this to `2h` instead of `2m`?